skyvern/library/skyvern.py

```diff
@@ -42,17 +45,34 @@ def __init__(
         follow_redirects: bool | None = True,
         httpx_client: httpx.AsyncClient | None = None,
     ) -> None:
+        env_base_url = ""
+        if base_url is None:
+            if os.path.exists(".env"):
+                load_dotenv(".env")
+            env_base_url = (os.getenv("SKYVERN_BASE_URL") or "").strip()
```

Comment:
let's use `settings.SKYVERN_BASE_URL`?
```suggestion
            env_base_url = (settings.SKYVERN_BASE_URL or "").strip()
```